routes/publish.js

```diff
@@ -11,9 +11,10 @@ const config = require('../config')
 // npm publish
 r.put('/:name', middleware.auth, function * () {
   let pkg = yield parse(this, {limit: '100mb'})
+  let tag = Object.keys(pkg['dist-tags'])[0]
```

Comment:
@dickeyxxx I am assuming that people pass multiple tags at once, should either throw an error here or handle if they pass multiples